### DISCUSSION TITLE
Postal auto start on system reboot

### DISCUSSION THREAD

**Discussion by mjml1982** [postalserver/postal#2087] [Help with using Postal]
Hello,

How we could autostart postal upon reboot?

Thanks.

**Comment by willpower232** [comment]
I think the easiest solution is to add the restart lines to the docker-compose.yml like this https://github.com/postalserver/install/pull/3

**Reply by mjml1982** [reply]
Thanks for the help, its not working after updating the docker-compose.yml.

**Reply by mjml1982** [reply]
It worked when I updated both docker-compose.yml files.

**Reply by willpower232** [reply]
yeah one is a template and one is the one actually being used
